plugins/ilert/src/components/Incident/IncidentActionsMenu.tsx

```diff
@@ -64,6 +64,9 @@ export const IncidentActionsMenu = ({
     try {
       handleCloseMenu();
       setProcessing(true);
+
+      const { userEntityRef } = await identityApi.getBackstageIdentity();
+      const { name: userName } = parseEntityRef(userEntityRef);
```

Comment:
Let me just mention that this code does assume that the `userEntityRef` never was set to just a shorthand ID (parseEntityRef will explode if it gets just a name and no kind/namespace defaults passed in). Maybe that's safe to assume at this point, but wanted to mention it anyway. The ultra careful version would be to pass `{ defaultKind: 'User'; defaultNamespace: 'default' }` as the second argument.

Reply:
Switched to the careful version, but also added a warning to the `AppIdentityProxy` in case an invalid `userEntityRef` is passed. It does feel pretty awkward but I don't have much better ideas tbh 😅